fe/src/app/dashboard/page.tsx

```diff
@@ -1,28 +1,28 @@
 import React from "react";
-import { getSession, getAccessToken } from "@auth0/nextjs-auth0";
+import { getSession } from "@auth0/nextjs-auth0";
 import { SERVICE_ADVERTRS_ROUTE } from "@/utils/urls";
+import { getAccessToken } from "@auth0/nextjs-auth0";
 
 const getServiceAdvertsData = async () => {
   const { accessToken } = await getAccessToken();
   if (!accessToken) {
     throw new Error(`Requires authorization`);
   }
-  // Fix 500 error, accessToken is available and obtained
   const res = await fetch(
     `${process.env.API_SERVER_URL}${SERVICE_ADVERTRS_ROUTE}`,
     {
       method: "GET",
       headers: {
         Authorization: `Bearer ${accessToken}`,
+        "Access-Control-Allow-Origin": "*",
       },
     }
   );
-  // console.log(res);
   if (!res.ok) {
     const json = await res.json();
 
     return {
-      text: json.message || res.statusText || "Unable to fetch message.",
+      text: json.message || res.statusText || "Unknown error occoured.",
```

Comment:
Maybe move `Unknown error occoured.` to constants this message as we might used in multiple places accross aplication?